dags/pull_wys.py

```diff
@@ -93,7 +94,11 @@ def custom_fail_slack_alert(context: dict) -> str:
                         'api_key':api_key}
                         )
     
-    t2 = PythonOperator(
+    @task
+    def task_get_schedules():
+        get_schedules(wys_postgres, api_key)
```

Comment:
Same thing with pulling the `api_key` from the variable. That should happen inside the decorated function and not outside.